packages/builtin/stdin.pony

```diff
@@ -116,8 +116,11 @@ actor Stdin
         var data = recover Array[U8].undefined(chunk_size) end
         var again: Bool = false
 
-        let len = @pony_os_stdin_read[USize](data.cstring(), data.space(),
-          addressof again)
+        let len = @pony_os_stdin_read[USize](
+          data.cpointer(),
+          data.space(),
+          addressof again
+        )
```

Comment:
This is probably the style I would use on my personal code, but I think it doesn't match the Pony standard library style of only using line breaks where strictly necessary in argument and parameter lists.